packages/firebase_crashlytics/darwin/Classes/FirebaseCrashlyticsPlugin.m

```diff
@@ -41,37 +41,32 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     // Add logs.
     NSArray *logs = call.arguments[@"logs"];
     for (NSString *log in logs) {
-      // Here and below, use CLSLog instead of CLS_LOG to try and avoid
-      // automatic inclusion of the current code location. It also ensures that
-      // the log is only written to Crashlytics and not also to the offline log
-      // as explained here:
-      // https://support.crashlytics.com/knowledgebase/articles/92519-how-do-i-use-logging
-      CLSLog(@"%@", log);
```

Comment:
It has been a while since I wrote this, so I am not sure if it still applies. However, are you certain that this is not needed anymore (i.e. did you test it)?

Reply:
See https://github.com/flutter/plugins/pull/1912.

Reply:
I replaced it with that line:
https://github.com/FirebaseExtended/flutterfire/pull/2288/files/5de1034c898e4413ad478544fd4f8f5f45a75776#diff-c9c5c34361172b6bb1ad95a77b516649R44

Reply:
I was wondering if that does not have the problem I mentioned there.

Reply:
From the [migration guide](https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=ios): 

>  the new methods no longer print to stdout or NSLog (we recommend writing a wrapper if you want to keep this behavior).